qa/tasks/mgr/dashboard/test_perf_counters.py

```diff
@@ -65,9 +65,7 @@ def test_perf_counters_not_found(self):
         self.assertStatus(404)
         schema = JObj(sub_elems={
             'status': str,
-            'version': str,
             'detail': str,
-            'traceback': str,
```

Comment:
I suggest to use `allow_unknown=True` here, otherwise the test will fail if debugging is enabled.

Reply:
Thanks a lot for the tip, @votdev. And not only for that issue, but also the `request_id` would fail otherwise.